core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/telemetry/TelemetryPayload.scala

```diff
@@ -0,0 +1,15 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream.telemetry
+
+// iglu:com.snowplowanalytics.oss/oss_context/jsonschema/1-0-1
+private case class TelemetryPayload(
```

Comment:
This class is used only to create the `Json` with `.asJson`. We can get rid of it and create it directly in `makeHeartbeatEvent`, like [here](https://github.com/snowplow/enrich/blob/feature/file-sink-rotate/modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Telemetry.scala#L101-L112)

Reply:
I think it is good practice to have a class just for the single purpose of serializing it. Mostly for documentation reasons, and for theoretical flexibility in case we start to do more things with it. 

There is a single call to toJson, while an example has toJson call for every field.

Reply:
I think both methods are valid, and both sides of the argument are valid too..  I've done it both ways in the past.

I'm even OK with being inconsistent between collector and enrich.  Because in this case the inconsistency does not affect the inputs/outputs of the application.